File: basic/Class_Member/2.py
class Cs :
    # 장식자 붙여주면 됀다.
    @staticmethod
    def static_method() :
        print("Static method")
    # class method에는 cls 가져야 한다.
    @classmethod
    def class_method(cls) :
        print("Class method")
    def instance_method(self) :
        print("Instance method")

i = Cs()
Cs.static_method()
Cs.class_method()
i.instance_method()

File: basic/Class_Member/2.rb
class Cs
  # 클래스 멤버
  # 앞에 클래스 이름을 붙여야 한다.
  def Cs.class_method()
    p "Class method"
  end
  def instance_method()
    p "Instance method"
  end
end

i = Cs.new()
Cs.class_method()
i.instance_method()
# i.class_method()
# Cs.instance_method()
